department-of-veterans-affairs/next-build
# PR: RSVP Capitalization #667, Intro Text Styling #661, and Social Share Styling #662

# Description
1) Resolves the RSVP Capitalization 
2) Fix the margin issue for #661 
3) Resolve these styling issues for #662 
- The VADS margin class on Cost should be 1 instead of 2. This causes the spacing to be too big between this and whatever item is below
- On social links in prod there is no padding to the right, in dev there is 20px of padding to the right. Remove class "medium-screen:vads-u-margin-right--2p5"
- Padding between icon and link in prod is 7px and 4px in dev
- Space between X link and Register link is too small

## Ticket
Relates to #[667](https://github.com/department-of-veterans-affairs/next-build/issues/667). (or closes?)
Relates to #[661](https://github.com/department-of-veterans-affairs/next-build/issues/661). (or closes?)
Relates to #[662](https://github.com/department-of-veterans-affairs/next-build/issues/662). (or closes?)

## Notes
There is a ghost 3px padding/margin between the va-icon and va-link for the social share that is unidentified what is applying the extra spacing

## Testing Steps

- Use Inspect Chrome Developer Tools to compare the elements and wrapping between local and prod.
- Compare differences on Tugboat
- Compare on event listing page and an event page
- Click on links on breadcrumbs to see if the redirect to the same pages
- Examine and compare next-build and content-build code to have matching modifiers and conditions

## QA steps
What needs to be checked to prove this works?  

- Padding and spacing should be the same
- Style should be matching (bold, underline, icons)
- Redirects work

What needs to be checked to prove it didn't break any related things?  
- Redirects should be checked but I don't believe anything else is impacted.
What variations of circumstances (users, actions, values) need to be checked?
- N/A

## Screenshots
Before: 
Dev and Local
<img width="1719" alt="Screenshot 2024-07-30 at 11 26 48 AM" src="https://github.com/user-attachments/assets/b235e72c-6982-4cad-a9ba-bbf277095812">

After:
Dev and Prod showing they match
<img width="2056" alt="Screenshot 2024-08-28 at 10 31 05 AM" src="https://github.com/user-attachments/assets/6eb26008-44d1-4aae-967e-733c6b3c6750">


# Reviewer
<details>
<summary>Expand</summary
### Reviewing a PR

This section lists items that need to be checked or updated when making changes to this repository. 

## Standard Checks

```[tasklist]
```

## Merging an Approved Layout

When merging a layout, you must ensure that the content type has been turned on for `next-build` inside the `CMS`. This CMS flag must be turned on for editors to preview their work using the next build preview server.

Resource types (layouts) that have not been approved by design should NOT be pushed to production. Ensure that [slug.tsx](../src/pages/[[...slug]].tsx) does not include your resource type if it is not approved.

The status of layouts should be kept up to date inside [templates.md](./templates.md). This includes QA progress, development progress, etc. A link should be provided for where testing can occur.

## Merging a Non-Approved Layout

Your new resource type should not be included inside [slug.tsx](../src/pages/[[...slug]].tsx). Items added here will go into production once merged into the `main` branch. It is imperative that we do not push anything live that has not been approved.

Ensure that this layout has been added to the [templates.md](./templates.md) file with the current status of the work.
</details>
